roles/taca/templates/taca_site_config.yml.j2

```diff
@@ -0,0 +1,16 @@
+log:
+    log_file: "{{ ngi_pipeline_site_path }}/taca.log"
+
+cleanup:
+    irma:
+        flowcell:
+            root: 
+                - "{{ proj_root }}/{{ ngi_pipeline_sthlm_delivery }}/incoming"
```

Comment:
If you're referring to "site" I figured I'd make the template generic then just omit the uppsala creation.